resources/vip-template/VIP/Scene.xctemplate/UIViewController/___FILEBASENAME___ViewController.swift

```diff
@@ -9,25 +9,20 @@
 import UIKit
 
 protocol ___VARIABLE_sceneName___ViewProtocol: class {
-    var vc: ___VARIABLE_sceneName___ViewController? { get }
-    var interactor: ___VARIABLE_sceneName___InteractorProtocol? { get set }
-    var router: ___VARIABLE_sceneName___RouterProtocol? { get set }
-
-    // Replaced by interactor's output functions and conform the protocol in presenter
+    func bind(to: ___VARIABLE_sceneName___InteractorProtocol)
+    func dismiss(animated: Bool, completion: (() -> Void)?)
```

Comment:
This is here to allow the router to dismiss this VC without knowing that it is related to UIKit

Reply:
The only issue with having a generic dismiss method is that there may be cases where the scene is not dismissible (it only pops), or where it's the root of the app so we shouldn't have a dismiss method. Do you think these are serious cases to consider?